commit ca1b9fffa5284722e9d86ce22fff3dd1a38f77ce
Author: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>
Date:   Sat Aug 23 12:40:07 2025 +0200
Upstream: https://github.com/pololu/pololu-tic-software/pull/13

    Fix detection of system libtinyxml2
    
    * Add missing letter `Y` in `LIBTINYXML2`.
    * Add linker flags

--- a/bootloader/CMakeLists.txt
+++ b/bootloader/CMakeLists.txt
@@ -5,7 +5,7 @@ string (REPLACE ";" " " LIBUSBP_CFLAGS_STR "${LIBUSBP_CFLAGS}")
 string (REPLACE ";" " " LIBUSBP_LDFLAGS_STR "${LIBUSBP_LDFLAGS}")
 
 if (USE_SYSTEM_LIBTINYXML2)
-  pkg_check_modules(LIBTINXML2 REQUIRED tinyxml2)
+  pkg_check_modules(LIBTINYXML2 REQUIRED tinyxml2)
   STRING(REPLACE ";" " " LIBTINYXML2_CFLAGS_STR "${LIBTINYXML2_CFLAGS}")
   STRING(REPLACE ";" " " LIBTINYXML2_LDFLAGS_STR "${LIBTINYXML2_LDFLAGS}")
 else ()
@@ -28,5 +28,5 @@ set_property (TARGET bootloader PROPERTY
 set_property (TARGET bootloader PROPERTY
   INTERFACE_COMPILE_OPTIONS ${LIBUSBP_CFLAGS})
 
-target_link_libraries (bootloader "${LIBUSBP_LDFLAGS_STR}")
+target_link_libraries (bootloader "${LIBUSBP_LDFLAGS_STR}" "${LIBTINYXML2_LDFLAGS_STR}")
 
